source/fundamentals/connection.txt

```diff
@@ -103,11 +100,13 @@ redundancy and high data availability.
 
 To connect to a replica set deployment, specify the hostname (or IP address)
 and port numbers of one or more members of the replica set separated by commas.
-Use the  to specify the name
-of the replica set with which your application is connecting. When the
-replica set option is specified, the driver assumes that your
-connection string uses a seed list of multiple hosts instead of a single
-``host:port`` configuration.
+By default, specifying the hostname and port number of a single MongoDB
+instance will cause the driver to only connect to the MongoDB instance provided.
+However, if you specify the name of the replica set using the
+``replicaSet`` parameter, the driver creates a replica set connection
+and will discover all of the members of the set. Specifying multiple hosts
+also causes the driver to create a replica set connection and discover
+all members of the set.
```

Comment:
Specifying `directConnection=false` will also do it.  Can you add that as the preferred alternative?

Reply:
Created bullet points detailing the three known ways of discovering additional hosts in a replica set, with "specify multiple hosts" as the last option.